docs/DeveloperGuide.md

```diff
@@ -1,29 +1,173 @@
 # Developer Guide
 
-## Acknowledgements
+## Design & Implementation
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+### Ui
+The `Ui` class handles user input and basic output messages. It can
+* Read user input
+* Output various messages to the user
 
-## Design & implementation
+![img.png](imgs/UiClass.png)
```

Comment:
Note that this is non-standard UML notation. You can use the info here to change settings for PlantUML: https://github.com/nus-cs2113-AY2122S1/forum/issues/114

![image](https://user-images.githubusercontent.com/53231968/198247059-c6b67337-0684-4302-9f39-ac56005a1597.png)